NEWS.md

```diff
@@ -1,3 +1,7 @@
+3.15.12 (05/07/2017)
+-------
+* Same version number, no code changes. Doc enhancement only.
```

Comment:
ok, just following the best practices recommended in all the code repos so the build doesn't fail :-)